models.py

```diff
@@ -621,14 +624,44 @@ def on_model_change(self, form, model, is_created):
             msg.html = 'Thank you {}. <br/><br/><p>Please click <a href="{}/admin/reset">reset</a> and enter the registered email to reset your password.</p>'.format(model.merchant_name, app.config["SITE_URL"])
             mail.send(msg)
 
-    column_list = ['merchant_name', 'merchant_code', 'email', 'roles', 'max_settlements_per_month', 'merchant_rate', 'customer_rate', 'wallet_address']
+    column_list = ['merchant_name', 'merchant_code', 'roles', 'email', 'max_settlements_per_month', 'merchant_rate', 'customer_rate', 'wallet_address']
     column_editable_list = ['merchant_name', 'roles', 'max_settlements_per_month', 'merchant_rate', 'customer_rate']
     form_columns = ['roles', 'merchant_name', 'email']
     form_args = dict(
         email=dict(validators=[DataRequired(), validate_email_address]),
         merchant_name=dict(validators=[DataRequired()])
     )
 
+class NoneAdminUserModelView(RestrictedModelView):
```

Comment:
Change this to have AdminUserModelView as its parent and then we can reduced duplicated code

Also change class name to FinanceUserModelView

Reply:
done